tests/test-cases/Group6-VIC-Machine/6-16-Config.robot

```diff
@@ -18,9 +18,45 @@ Resource  ../../resources/Util.robot
 Suite Setup  Install VIC Appliance To Test Server
 Suite Teardown  Cleanup VIC Appliance On Test Server
 
+*** Keywords ***
+Verify VCH Debug State
+    [Arguments]  ${expected}
+    ${rc}  ${output}=  Run And Return Rc And Output  govc vm.info -e %{VCH-NAME} | grep guestinfo.vice./init/diagnostics/debug
```

Comment:
this keyword can be more generic to accept grep parameter, to make it useful for other attribute